API.md

```diff
@@ -285,3 +321,39 @@ If multiple added/omits are present, the <code>add/omit</code> symbol is only pr
     </tr>  </tbody>
 </table>
 
+<a name="AltIntervals"></a>
+
+## AltIntervals : <code>Object</code>
+Intervals affected by the Alt modifier when parsing an altered chord written "C7alt", for example.
+
+**Kind**: global typedef  
+**Properties**
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th><th>Type</th><th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+<tr>
+    <td>fifthFlat</td><td><code>Boolean</code></td><td><p>if the alt modifier should flatten the fifth</p>
```

Comment:
more like "if the alt modifier should yield/give/return/render a flat five/flattened fifth"

With `flatten the fifth`, it looks like you take the 5th and flatten it. Which is weird if fifthSharp is also true. (you can't both flatten and sharpen the same note at the same time)

Reply:
Good catch 👍